src/SelfScreener/HowAreYouFeeling.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { FunctionComponent } from "react"
+import { useTranslation } from "react-i18next"
+import { useNavigation } from "@react-navigation/native"
+import {
+  Image,
+  ImageSourcePropType,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from "react-native"
+
+import { SelfScreenerStackScreens } from "../navigation"
+import { Typography, Spacing, Iconography, Colors, Outlines } from "../styles"
+import { GlobalText } from "../components"
+import { Images } from "../assets"
+
+const HowAreYouFeeling: FunctionComponent = () => {
+  const { t } = useTranslation()
+  const navigation = useNavigation()
+
+  const handlePressGood = () => {
+    navigation.navigate(SelfScreenerStackScreens.AsymptomaticFlowIntro)
+  }
+  const handlePressNotWell = () => {
+    navigation.navigate(SelfScreenerStackScreens.EmergencySymptomsQuestions)
+  }
+
+  return (
+    <View style={style.contentContainer}>
+      <GlobalText style={style.headerText}>
+        {t("self_screener.how_are_you_feeling.title")}
+      </GlobalText>
+      <SelectFeeling
+        handleOnPressGood={handlePressGood}
+        handleOnPressNotWell={handlePressNotWell}
+      />
+    </View>
+  )
+}
+
+interface SelectFeelingProps {
+  handleOnPressGood: () => void
+  handleOnPressNotWell: () => void
+}
+
+const SelectFeeling: FunctionComponent<SelectFeelingProps> = ({
```

Comment:
Since this page is so simple I would opt to inline this instead of creating a component and having to thread through the props. Probably makes sense to keep the `FeelingButton` component though, that feels useful because of the duplication.